pkg/serverless/serverless.go

```diff
@@ -40,6 +40,8 @@ const (
 	requestTimeout     time.Duration = 5 * time.Second
 	clientReadyTimeout time.Duration = 2 * time.Second
 
+	safetyBufferTimout time.Duration = 100 * time.Millisecond
```

Comment:
```suggestion
	safetyBufferTimeout time.Duration = 100 * time.Millisecond
```
Also, does that mean if the handler completes <100ms before timeout, we will force it to be a timeout instead?